### DISCUSSION TITLE
AVCHD Capability

### DISCUSSION THREAD

**Discussion by jc-frosty** [justdan96/tsMuxer#776] [Q&A] [answered]
Is this able to make AVCHD folders that can be written to a disk or drive for use with a blu-ray player?  Could this be used as a substitute for MultiAVCHD based on this [posts](https://www.toyotanation.com/threads/rear-seat-entertainment-system-sd-card-solved.985674/) usage

**Comment by donnafaye0323** [comment, marked as answer]
Yes. It will be a great substitute since is it for Windows Only.

**Reply by jc-frosty** [reply]
Thanks.

**Reply by donnafaye0323** [reply]
I now use MultiACHD, AVISynth and K-Lite Codec Pack-the three tools running on CrossOver on Linux-this project helps a lot for creating Blu-rays with a stylish look. Have fun!!

**Comment by donnafaye0323** [comment]
Also, it's better for importing PVR video files for Blu-ray authoring. Here's a snapshot of a MakeBD program that is No longer developed:

![05-makebd_2](https://github.com/justdan96/tsMuxer/assets/40868495/4a9106df-9e92-4ae5-8061-656ea0d7ba50)

This is why you use it for AVCHD and Blu-ray creation. Thank you for you information.

**Comment by donnafaye0323** [comment]
I can also make Blu-ray videos with MultiAVCHD and AVISynth running under Linux with CrossOver and Wine, and it now works very well. You should have MultiAVCHD 4.1 and AviSynth installed on Wine to make the menu-authoring function work. Just install them on Wine and-Viola it goes great!!
